CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v4.1.8
```

Comment:
Shouldn't this be a minor bump instead, since #1180 brings a new feature to the release? Patch bump in this case does not meet semver standards.

Reply:
Great point!

https://github.com/actions/checkout/pull/1878/commits/10878643c9627c4949c74bb765408ee47e76533b